docs/api/upgrade-assistant/status.asciidoc

```diff
@@ -11,13 +11,19 @@ Check the status of your cluster.
 [[upgrade-assistant-api-status-request]]
 ==== Request
 
-`GET <kibana host>:<port>/api/upgrade_assistant/status`
+`GET <kibana host>:<port>/api/upgrade_assistant/status?targetVersion=9.0.0`
+
+`targetVersion`::
+(optional, string): Version to upgrade to.
 
 [[upgrade-assistant-api-status-response-codes]]
 ==== Response codes
 
 `200`::
   Indicates a successful call.
+  
+`403`::
```

Comment:
New response code to handle the case where the target version to upgrade to is not supported, for example, upgrading more than 1 major or downgrading.